pkg/ovs/ovn-nbctl-legacy.go

```diff
@@ -622,21 +655,33 @@ func (c *LegacyClient) SnatRuleExists(eip, ipCidr string) (bool, error) {
 	return false, fmt.Errorf("snat rule not exist")
 }
 
-func (c LegacyClient) DeleteMatchedStaticRoute(cidr, nexthop, router string) error {
+func (c LegacyClient) DeleteMatchedStaticRoute(cidr, nexthop, router, routeTable string) error {
+	if routeTable == util.MainRouteTable {
```

Comment:
if MainRouteTable is "" in the ovn nb, how about set MainRouteTable = "" in the const.go

Reply:
> if MainRouteTable is "" in the ovn nb, how about set MainRouteTable = "" in the const.go

good idea, already changed